ghproxy/ghcache/ghcache.go

```diff
@@ -170,6 +174,10 @@ func (u upstreamTransport) RoundTrip(req *http.Request) (*http.Response, error)
 	if etag != "" {
 		resp.Header.Set("X-Conditional-Request", etag)
 	}
+
+	ghmetrics.GithubTokenMetrics(resp.Header, reqStartTime)
```

Comment:
Could we add a verb to these function names? What do they actually do? PublishMetrics?

Reply:
done